tests/unit/lms/services/grouping_test.py

```diff
@@ -73,42 +73,22 @@ def generate_authority_provided_id(self, svc):
 
 
 class TestUpsertWithParent:
-    CONTEXT_ID = "context_id"
-    TOOL_CONSUMER_INSTANCE_GUID = "t_c_i_guid"
```

Comment:
It's nice to be rid of these (they moved into the `partial` below): I think these kinds of `CONSTANTS` on test classes are ugly and distracting